python/dask_cudf/dask_cudf/core.py

```diff
@@ -643,7 +643,25 @@ def reduction(
     return dd.core.new_dd_object(dsk, b, meta, (None, None))
 
 
-from_cudf = dd.from_pandas
+def from_cudf(data, npartitions=None, chunksize=None, sort=True, name=None):
+    if isinstance(getattr(data, "index", None), cudf.MultiIndex):
+        raise NotImplementedError(
+            "dask_cudf does not support MultiIndex Dataframes."
+        )
```

Comment:
Does it make sense long term  to add a dispatch for a `MultiIndex` class into Dask instead of handling this directly in `dask-cudf`?

Reply:
Sure - We may get support for something like that if there are enough cases of hard-coded `pd.MultiIndex` objects across the dask.dataframe codebase.  However, there is definitely a general resistence to adding dispatch functions unless completely necessary.  Since we are wrapping `from_pandas` here, this particular example might not be the best argument.

Reply:
I guess my fear is there's coding floating out there and being promoted that shows using `dd.from_pandas(cudf.DataFrame(...))` where it will silently cause issues due to checking only `pd.MultiIndex`.

Reply:
Your fear of silent issues from hard-coded pandas objects is completely warranted.  I will try to search through dask for evidence that we need to add a dispatch mechanism for MultiIndex.   Either way, I definitley think we should be discouraging people from using `from_pandas` for cudf objects in the future :)